sdk/cosmos/azure-cosmos/azure/cosmos/_retry_utility.py

```diff
@@ -210,6 +211,15 @@ def send(self, request):
                 timeout_error.response = response
                 timeout_error.history = retry_settings['history']
                 raise
+            except ServiceRequestError as err:
```

Comment:
Looks good to me. Do we have test coverage for service request error retries?

Reply:
adding those now